webapp/tests/Unit/Controller/Jury/ImportExportControllerTest.php

```diff
@@ -21,9 +21,6 @@ public function testIndexBasic() : void
             self::assertSelectorExists(sprintf('h2:contains("%s")', $section));
         }
         self::assertSelectorExists('small:contains(\'Create a "Web Services Token"\')');
-
-        // We've reached the end of the page.
-        self::assertSelectorExists('li:contains("scoreboard.tsv")');
```

Comment:
As the comment says the intention was so find out if we see some item near the end of the page so we have some basic certainty that the whole thing is rendered. So I would not remove it but put an assertion for a thing near it in its place.

Reply:
@vmcj I did push a commit to address this now. Let's merge this?

Reply:
Let me fixup my things, and then I'll merge it yes.